### PR TITLE
update nuget to rtm-2259

### PR DESCRIPTION
**Pull request by rohit21agrawal**
update nuget to 4.0.0-rtm-2259

CC: @rrelyea @dsplaisted @srivatsn

### CODE REVIEW

**Comment by srivatsn** [comment]
Tagging @MattGertz - regular NuGet update of fixes that went into VS yesterday.

**Comment by natidea** [comment]
Approved to merge via [378318](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/378318)

**Comment by rohit21agrawal** [comment]
@srivatsn @natidea can't merge until someone approves the pull request

**Review by srivatsn — APPROVED** [review]

**Pull request merged**
